runelite-mixins/src/main/java/net/runelite/mixins/RSClientMixin.java

```diff
@@ -1820,4 +1851,28 @@ public static void changeGameOptions(int var0)
 			client.getCallbacks().post(VolumeChanged.class, volumeChanged);
 		}
 	}
+
+	@Replace("getWidgetClickMask")
+	public static int getWidgetClickMask(Widget widget)
+	{
+		IntegerNode integerNode = (IntegerNode) client.getWidgetFlags().get(((long) widget.getId() << 32) + (long) widget.getIndex());
+
+		int widgetClickMask;
+
+		if (integerNode == null)
+		{
+			widgetClickMask = widget.getClickMask();
+		}
+		else
+		{
+			widgetClickMask = integerNode.getValue();
+		}
+
+		if (allWidgetsAreOpTargetable)
+		{
+			widgetClickMask |= 2097152;
```

Comment:
may be better to use the api class, so it makes more sense than 2097152

(if not api class, 0b1000000000000000000000 or 0x200000 maybe?)